src/lang-promql/complete/hybrid.ts

```diff
@@ -284,16 +262,20 @@ export class HybridComplete implements CompleteStrategy {
 
     for (const completionList of data) {
       for (const label of completionList.labels) {
-        const completionResult = this.filter(escape(label), text, label, context);
-        if (completionResult) {
+        const completionResult = context.filter(
+          { label: label, original: label, apply: label, type: completionList.type, score: 0 },
+          text
+        );
+        console.log('==', completionResult);
```

Comment:
I guess this should be emoved. Quite weird eslint doesn't yell at it

Reply:
Whoops, thanks, fixed! :)